Description: Builders don't have access to network, skip this test
Author: Vincent Bernat <bernat@debian.org>
Forwarded: no

Index: cookiecutter-0.6.4/tests/test_vcs.py
===================================================================
--- cookiecutter-0.6.4.orig/tests/test_vcs.py	2013-09-29 13:00:09.586944734 +0200
+++ cookiecutter-0.6.4/tests/test_vcs.py	2013-09-29 13:01:05.696038267 +0200
@@ -33,6 +33,7 @@
 
 class TestVCS(unittest.TestCase):
 
+    @unittest.skip("disabled, no network")
     def test_git_clone(self):
         repo_dir = vcs.git_clone(
             'https://github.com/audreyr/cookiecutter-pypackage.git'
@@ -50,6 +51,7 @@
             shutil.rmtree('cookiecutter-pypackage')
         os.mkdir('cookiecutter-pypackage/')
 
+    @unittest.skip("disabled, no network")
     @patch(input_str, lambda: 'y')
     def test_git_clone_overwrite(self):
         if not PY3:
@@ -60,6 +62,7 @@
         self.assertEqual(repo_dir, 'cookiecutter-pypackage')
         self.assertTrue(os.path.isfile('cookiecutter-pypackage/README.rst'))
 
+    @unittest.skip("disabled, no network")
     @patch(input_str, lambda: 'n')
     def test_git_clone_cancel(self):
         if not PY3:
Index: cookiecutter-0.6.4/tests/test_main.py
===================================================================
--- cookiecutter-0.6.4.orig/tests/test_main.py	2013-08-21 12:33:41.000000000 +0200
+++ cookiecutter-0.6.4/tests/test_main.py	2013-09-29 13:01:55.521009341 +0200
@@ -66,6 +66,7 @@
 
 class TestCookiecutterRepoArg(unittest.TestCase):
 
+    @unittest.skip("disabled, no network")
     @patch(input_str, lambda x: '')
     def test_cookiecutter_git(self):
         if not PY3:
